scapy/contrib/modbus.py

```diff
@@ -14,11 +14,9 @@
 # You should have received a copy of the GNU General Public License
 # along with Scapy. If not, see <http://www.gnu.org/licenses/>.
 
-# Copyright (C) 2016 Arthur Gervais, Ken LE PRADO, Sébastien Mainand
+# Copyright (C) 2016 Arthur Gervais, Ken LE PRADO, Sébastien Mainand
 
-from scapy.packet import *
-from scapy.fields import *
-from scapy.layers.inet import *
+from scapy.all import *
```

Comment:
I do my test again and no it's not useful. The old imports are only what we need. I missed this.